### PR TITLE
Feed context to memecomplete.com on copy

### PR DESCRIPTION
**Pull request by jacebrowning**
Only when the user clicks the copy button do we know we have a complete image URL.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 out
 .lein-repl-history
 .nrepl-port
+.sass-cache
```
Comment (jasich) [inline .gitignore:9]: I love when random `.` files/directories show up for some and not others.

**Review by jasich — COMMENTED** [review]
My only thought is that you may want to remove the call to `api/meme-complete` from the current location https://github.com/jasich/mighty-fine-memes/blob/327815fb661ee7dd1e6708227f53483c0a58654f/src/cljs/memegen_ui/events.cljs#L122  and just rely on it's use during copy.  Is that the intent?

**Comment by jacebrowning** [comment]
Yeah, the intention was to only call the API once. I just force pushed.

**Review by jasich — APPROVED** [review]

**Comment by jacebrowning** [comment]
Does this deploy automatically?

**Comment by jasich** [comment]
Sorry, it get's deployed infrequently so I just do it manually.  I just pushed it.

**Comment by jacebrowning** [comment]
screen shot 2017-06-29 at 1 46 45 pm

[memecomplete](http://memecomplete.com/?share=https%3A//memegen.link/custom/hooked_up_the_api/it%2527s_working%2521_it%2527s_working%2521.jpg%3Falt%3Dhttp%3A//i.imgur.com/ZVIR76A.jpg)

**Pull request merged**